src/groupeddataframe/groupeddataframe.jl

```diff
@@ -64,18 +64,24 @@ into row groups.
   if `sort=nothing` (the default) then the fastest available grouping algorithm
   is picked and in consequence the order of groups in the result is undefined
   and may change in future releases; below a description of the current
-  implementation is provided.
+  implementation is provided. Additionally `sort` can be a `NamedTuple` having
+  some or all of `alg`, `lt`, `by`, `rev`, and `order` fields. In this case
+  the groups are sorted and their order follows the [`sortperm`](@ref) order.
 - `skipmissing` : whether to skip groups with `missing` values in one of the
   grouping columns `cols`
 
 # Details
+
 An iterator over a `GroupedDataFrame` returns a `SubDataFrame` view
 for each grouping into `df`.
 Within each group, the order of rows in `df` is preserved.
 
 `cols` can be any valid data frame indexing expression.
 In particular if it is an empty vector then a single-group `GroupedDataFrame`
-is created.
+is created. As a special case, if a list of columns to group by is passed
```

Comment:
How about putting this in the arguments list instead?

Reply:
Makes sense - I moved the whole passage.